src/main/java/de/slimecloud/slimeball/features/alerts/youtube/YoutubeListener.java

```diff
@@ -0,0 +1,33 @@
+package de.slimecloud.slimeball.features.alerts.youtube;
+
+import de.cyklon.jevent.EventHandler;
+import de.cyklon.jevent.Listener;
+import de.slimecloud.slimeball.main.SlimeBot;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.channel.concrete.TextChannel;
+
+@Slf4j
+@Listener
+@RequiredArgsConstructor
+public class YoutubeListener {
+	private final SlimeBot bot;
+
+	@EventHandler
+	public void onUpload(YoutubeVideoEvent event) {
+		bot.getJda().getGuilds().forEach(g -> bot.loadGuild(g).getYoutube().ifPresent(c -> {
+			TextChannel channel = bot.getJda().getTextChannelById(c.getChannel());
+
+			if (channel != null) {
+				Role role = bot.getJda().getRoleById(c.getRole());
```

Comment:
getRole in YoutubeConfig should already return role (same for getChannel), see all other guild confiugs